tests/stub_virtual_machine_factory.h

```diff
@@ -21,13 +21,17 @@
 #include "stub_virtual_machine.h"
 #include "stub_vm_image_vault.h"
 
-#include <multipass/virtual_machine_factory.h>
+#include <multipass/exceptions/not_implemented_on_this_backend_exception.h>
+#include <multipass/network_interface_info.h>
+#include <multipass/path.h>
+
```

Comment:
No. However, I needed to include `multipass/network_interface_info.h` in `mock_virtual_machine_factory.h` when I removed it from here (and I left the include there because it makes more sense than here).